### PR TITLE
logout , restricted words and authUser fixes

### PR DESCRIPTION
**Pull request by rarmitag**
*Issue #:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by katiemcgoff** [comment]
@rarmitag There are a lot of conflicts with findFilter.vue again. Can you merge in the upstream master to your branch?

File: client/package.json [hunk 1]
```diff
@@ -65,6 +65,7 @@
     "postcss-import": "^11.0.0",
     "postcss-loader": "^2.0.8",
     "postcss-url": "^7.2.1",
+    "react-test-renderer": "^16.4.1",
```
Comment (katiemcgoff) [inline client/package.json]: What is this for?

File: client/static/keycloak/keycloak.json [hunk 2]
```diff
@@ -2,9 +2,9 @@
   "realm": "nest",
   "auth-server-url": "https://dev-sso.pathfinder.gov.bc.ca/auth",
   "ssl-required": "external",
-  "resource": "namex-TEST",
+  "resource": "namex-DEV",
```
Comment (katiemcgoff) [inline client/static/keycloak/keycloak.json]: Should this file be changed from test to dev keycloak? Does this get overridden in the server environments (in which case it doesn't matter)?

**Comment by rarmitag** [comment]
This is for jest test

**Comment by rarmitag** [comment]
Jest testing copy that I didn’t want to lose, should have put it elsewhere.

**Comment by rarmitag** [comment]
Yes, it is overridden in the server environment so is only useful locally

**Pull request merged**